### ISSUE TITLE
Doedre's Skin Reduced Curse Effectiveness isn't applied

### ISSUE THREAD

**Issue by pundm** [PathOfBuildingCommunity/PathOfBuilding#2194]
* **What happened?**
I wanted to calculate my curse effectiveness using Doedre's Skin body armor. PoB currently does not seem to apply the reduced effect of curses.

Quote from the PoE wiki:
"The body armour grants Summon Doedre's Effigy, as well as support the socketed Hex skill gem with SupportCursePillarTriggerCurses. That support has 25% reduced Effect of Supported Curses."

* **What steps will reproduce the bug?**
In PoB: Equip Doedre's Skin and socket a Curse in it. 

* **Does it reproduce every time?**
Yes

* **Provide a build code:**
https://pastebin.com/18QNkhqU

* **Screenshots:**
https://imgur.com/a/QBkaSZg

In this case, PoB should show a multiplier of 2.54

**Comment by pundm** [comment]
Fixed when i add: 25% reduced effect of your Curses into the unique. I will try to make a pull request

**Comment by Regisle** [comment]
Yeah its not the unique Doedre's Skin which should apply that, its the skill Doedre's Effigy

**Comment by pundm** [comment]
I tried adding a line like "25% reduced effect of your curses" onto the chest, but it would reduce every curse gem's effectiveness, regardless if its socket-ed in the chest or not. Is there something like a local modifier expression in PoB? If so i could send a pull request. Still feels like a band-aid.

**Comment by ppoelzl** [comment]
Adding this mod to the item will not work for imported characters. You will have make changes to the skill it grants instead. Mods can be made to apply to a certain socket group only.

**Comment by pundm** [comment]
Nice this looks amazing. I just cloned the repository to give myself a try at it, but you were way faster. This gives me good inside into what to to next time. Thanks!
